unit-tests/live/frames/test-fps.py

```diff
@@ -102,10 +103,13 @@ def frame_cb(frame):
     cs = dev.first_color_sensor()
     # Set auto-exposure option as it might take precedence over requested FPS
     if product_line == "D400":
-        cs.set_option(rs.option.enable_auto_exposure, 1)
-        cs.set_option(rs.option.auto_exposure_priority, 0) # AE priority should be 0 for constant FPS
+        if cs.supports(rs.option.auto_exposure_priority):
+            cs.set_option(rs.option.enable_auto_exposure, 1)
+        if cs.supports(rs.option.auto_exposure_priority):
+            cs.set_option(rs.option.auto_exposure_priority, 0) # AE priority should be 0 for constant FPS
     elif product_line == "L500":
-        cs.set_option(rs.option.enable_auto_exposure, 0)
+        if cs.supports(rs.option.auto_exposure_priority):
+            cs.set_option(rs.option.enable_auto_exposure, 0)
```

Comment:
same

Reply:
Reverted the change to avoid conflicts with other L500 code changes in the repo